gitlab/v4/objects/merge_request_approvals.py

```diff
@@ -165,8 +165,6 @@ def set_approvers(
 
 
 class ProjectMergeRequestApprovalRule(SaveMixin, ObjectDeleteMixin, RESTObject):
-    _id_attr = "approval_rule_id"
-    _repr_attr = "approval_rule"
     id: int
     approval_rule_id: int
     merge_request_iid: int
```

Comment:
Seems like we could actually use `name` for the `_repr_attr` here from what i can tell based on the get&list response:
https://docs.gitlab.com/ee/api/merge_request_approvals.html#get-merge-request-level-rules

I see some of this is used later in the `save()` method to create a custom ID because of the mismatch, so I'd just like to take a quick look at what's happening there before I merge :)

Reply:
Maybe, but I don't think `name` could be used in a `get()` (which we do need to implement). I guess I'm not entirely sure when to use `_repr_attr` 😔

Reply:
I tried to document this a bit in https://github.com/python-gitlab/python-gitlab/pull/2068 (WIP).
`_repr_attr` will be used in repr methods if it's present in the object, e.g. after returning the response from the server. So I assume it will not be present on lazily created objects (and repr will show only id), but will be available after get/create/list etc. It is not used to actually fetch the resource.

Reply:
> Seems like we could actually use `name` for the `_repr_attr` here from what i can tell based on the get&list response: https://docs.gitlab.com/ee/api/merge_request_approvals.html#get-merge-request-level-rules

Done 🙂